bigquery/google/cloud/bigquery/dataset.py

```diff
@@ -26,6 +27,14 @@
 from google.cloud.bigquery.table import TableReference
 
 
+_PROJECT_PREFIX_PATTERN = re.compile(
+    r"""
```

Comment:
Why are we using a multi-line string here?

Reply:
Just for the readability. I think I’ll switch this to the single line, after correcting the pattern implementation.